src/java/org/apache/cassandra/utils/ObjectSizes.java

```diff
@@ -32,7 +32,6 @@
 public class ObjectSizes
 {
     private static final MemoryMeter meter = new MemoryMeter()
-                                             .omitSharedBufferOverhead()
```

Comment:
Looking through the usages of `measureDeep()`, it doesn't seem like any of those cases involve wrapping existing BBs, so this configuration option was indeed not necessary, but it would be good to get confirmation of that...